.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
```

Comment:
- [ ] I could not start your app sever since the start script is missing from your package.json, please add it and you will be going to go

<img width="300" alt="start" src="https://user-images.githubusercontent.com/57562869/154574710-eacd22d0-66c4-404e-b57c-2fb3443d1000.png">

- In case the error still persists, I would suggest you check this [link about webpack](https://webpack.js.org/concepts/#loaders)